js/index.js

```diff
@@ -1,7 +1,68 @@
 // Iteration 1: Names and Input
+var hacker1 = "Tarek";
+var hacker2 = "Uros";
+console.log(`The driver's name is ${hacker1}`);
+console.log(`The navigator's name is ${hacker2}`);
+// Iteration 2: Conditionals
+if (hacker1.length > hacker2.length){
+    console.log(`he driver has the longest name, it has ${hacker1.length} characters.`);
+}
+else if (hacker2.length > hacker1.length){
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`);
+}
+else{
+    console.log(`Wow, you both have equally long names, ${hacker2.length} characters!.`);
+}
 
+// Iteration 3: Loops
+var result1="";
+for (var i=0;i<hacker1.length;i++){
+    
+    result1=result1+" "+hacker1[i].toUpperCase();    
```

Comment:
To avoid to reapeat, you can use the operator += like the following example:
result1 += " "+hacker1[i].toUpperCase();